src/index.tsx

```diff
@@ -47,6 +49,13 @@ function Updaters() {
   )
 }
 
+if (isProductionEnv()) {
```

Comment:
What happens in non-production environments?
Will Sentry still log, or will `captureMessage` calls in other files be nops?

Reply:
A noop. I had it turned on by default initially in all environments with an appropriate tag, but that produces an unnecessary noise. Do we want to track in staging tho?